ratpack/src/main/groovy/com/baeldung/RatpackGroovyApp.groovy

```diff
@@ -0,0 +1,12 @@
+package com.baeldung;
+
+public class RatpackGroovyApp {
+
+    public static void main(String[] args) {
+        File file = new File("src/main/groovy/com/baeldung/Ratpack.groovy");
+        def shell = new GroovyShell()  
+        shell.evaluate(file)
```

Comment:
Why does this defer to a new groovy shell? Are we not already in a groovy runtime?

Reply:
We run this Application class while testing. We run it through jUnit test and Groovy Shell is required to run the Ratpack.groovy script to make it available as mock for testing.